CMakeLists.txt

```diff
@@ -44,6 +44,9 @@ endif()
 # project name
 project(SFML)
 
+# avoid missing libraries when building with latest NDK
+set(CMAKE_FIND_ROOT_PATH "${PROJECT_SOURCE_DIR};${CMAKE_FIND_ROOT_PATH}")
+
```

Comment:
As this [seems to only be](https://github.com/android/ndk/issues/912) an issue with the NDK (and potentially other cross-compilations), I'd suggest to put it inside an Android check.

Reply:
Updated.